# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1486546916 -3600
#      Wed Feb 08 10:41:56 2017 +0100
# Node ID 99ca557fd34f989717d08d56f22e30af6d9456d9
# Parent  af64b6b61db9a655c58f7abb245e0a8edd0a9163
maint: Use Octave whitespace conventions for scripts/general/inputParser.m.

* inputParser.m: Use Octave whitespace conventions.

diff --git a/scripts/general/inputParser.m b/scripts/general/inputParser.m
--- a/scripts/general/inputParser.m
+++ b/scripts/general/inputParser.m
@@ -269,26 +269,26 @@ classdef inputParser < handle
     Results       = struct ();
     Unmatched     = struct ();
     UsingDefaults = cell ();
   endproperties
 
   properties (Access = protected)
     ## Since Required and Optional are ordered, they get a cell array of
     ## structs with the fields "name", "def" (default), and "val" (validator).
-    Required   = cell ();
-    Optional   = cell ();
+    Required = cell ();
+    Optional = cell ();
     ## Parameter and Switch are unordered so we have a struct whose fieldnames
     ## are the argname, and values are a struct with fields "def" and "val"
-    Parameter  = struct ();
-    Switch     = struct ();
+    Parameter = struct ();
+    Switch    = struct ();
 
     ## List of Parameter and Switch names to ease searches
     ParameterNames = cell ();
-    SwitchNames     = cell ();
+    SwitchNames    = cell ();
 
     ## When checking for fieldnames in a Case Insensitive way, this variable
     ## holds the correct identifier for the last searched named using the
     ## is_argname method.
     last_name = "";
   endproperties
 
   properties (Access = protected, Constant = true)
